addons/sale/sale.py

```diff
@@ -140,6 +140,28 @@ def _get_invoiced(self, cr, uid, ids, field_name, arg, context=None):
                 res[order.id]['invoiced'] = True
         return res
 
+    def _amount_total(self, cr, uid, context=None):
+        vals = {}
+        invoiced = 0.0
+        paid = 0.0
+        if context is None:
+            context = {}
+        order_id = context.get('active_id')
+        if order_id:
+            order = self.browse(cr, uid, order_id, context=context)
+            for invoice in order.invoice_ids:
+                if invoice.state != 'cancel':
+                    invoiced += invoice.amount_total
+                if invoice.state == 'paid':
+                    paid += invoice.amount_total
+            symbol = order.currency_id.symbol
```

Comment:
It would be better
```python
amount_paid = (amount, symbol)
if position == 'before':
    amount_paid = amount_paid[::-1]

_('Including %s%s paid') % amount_paid